ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -97,6 +102,7 @@ class FaultInjector
   ros::Publisher m_fault_status_pub;
   ros::Publisher m_arm_fault_status_pub;
   ros::Publisher m_power_fault_status_pub;
+  ros::Publisher m_antennae_fault_status_pub;
```

Comment:
Just trying to understand the design better, why do we need to have a faults publisher per fault category?

Reply:
I suggest adding Terry as reviewer, as he wrote the original.  Perhaps some design decisions won't scale up well.  I'm not sure about this one.

Reply:
will add Terry